src/app/authentication/models/forgotPasswordModel.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type ForgotPasswordRequest struct {
+	Email         string `json:"email" validate:"required,email" example:"testUser@gmail.com" `
+	PanCardNumber string `json:"pancardNumber" validate:"required,len=10" example:"abgjhi6789" `
```

Comment:
Add custom validation for "PAN" field, wherein it contains alpha and num both. You can use regex for this